docs/aerodromes/classd/Moorabbin.md

```diff
@@ -22,48 +23,139 @@ MB ADC is responsible for the Class D airspace in the MB CTR `SFC` to `A025`.
-| BAW  | Join base 13R | Join base 17R | Join oblique downwind 31L | Join base 35L |
-| CARR | Join downwind 13R | Join downwind 17R | Join oblique base 31L | Join final 35L |
-| GMH  | Join downwind 13L | Join base 17L | Join oblique base 31R | Join base 35R |
-| ACE  | Join base 13L | Join oblique base 17L | Join base 31R | Join oblique downwind 35R |
+## VFR Inbound Procedures
 
-## Departures into CTA
-The overlying Class C airspace extends down to A025 to the north and A045 to the south. Aircraft planned into the overlying Class C airspace shall be issued an airways clearance by **MB ADC** and assigned `A050` or `RFL` if lower. These aircraft must be Next coordinated to the relevant TCU controller, who will assess the current traffic picture and provide a release when able. During times of peak TMA traffic, there may be extensive delays for departure and pilots may elect to depart directly into class G. See [Departure Coordination](#departures) for coordination requirements.
+Aircraft should report at the standard VFR entry points:  
+- **BTO** (to the north)  
+- **BAW** (to the west)  
+- **CARR** (to the southwest)  
+- **GMH** (to the southeast)  
+- **ACE** (to the northeast)  
+- **SHOAL** (to the southwest)
+
+| Entry Point | RWY 13 | RWY 17 | RWY 31 | RWY 35 |
+|-------------|--------|--------|--------|--------|
+| BTO         | Join final 13R | Join base 17R | Join downwind 31L | Join oblique downwind 35L |
+| BAW         | Join base 13R  | Join base 17R | Join oblique downwind 31L | Join base 35L |
+| CARR        | Join downwind 13R | Join downwind 17R | Join oblique base 31L | Join final 35L |
+| GMH         | Join downwind 13L | Join base 17L | Join oblique base 31R | Join base 35R |
+| ACE         | Join base 13L | Join oblique base 17L | Join base 31R | Join oblique downwind 35R |
+| SHOAL       | Join base 13L | Join base 17L | Join base 31R | Join base 35R |
+
+CTR Entry Altitude: `A010`.
 
 ## Circuits
-Circuits are to be flown at `A010`.
-
-| Runway | Day | Night |
-| ------ | ------ | ----|
-| 13L     | Left  | Left |
-| 13R     | Right | - |
-| 31L     | Left  | - |
-| 31R     | Right  | Right |
-| 17L     | Left  | Left |
-| 17R     | Right | - |
-| 35L     | Left  | - |
-| 35R     | Right  | Right |
 
-## Coordination
+### Fixed wing
+
+Circuit altitude: `A010`.
+
+| Runway  | Day Circuit Direction | Night Circuit Direction |
+|---------|------------------------|--------------------------|
+| 13L     | Left                  | Left                    |
+| 13R     | Right                 | N/A                     |
+| 31L     | Left                  | N/A                     |
+| 31R     | Right                 | Right                   |
+| 17L     | Left                  | Left                    |
+| 17R     | Right                 | N/A                     |
+| 35L     | Left                  | N/A                     |
+| 35R     | Right                 | Right                   |
+
+## Helicopters
```

Comment:
Worth noting that the helicopter operations are actually very similar to almost all Class D airports I've flown to in helicopters. So that isn't specific to per sae with Moorabbin. At least with the radio phraseology of calling airborne etc.

Reply:
> Worth noting that the helicopter operations are actually very similar to almost all Class D airports I've flown to in helicopters. So that isn't specific to per sae with Moorabbin. At least with the radio phraseology of calling airborne etc.

Absolutely taking no issue with how close to RL operations what you've done up is, it's all great. Unfortunately we do have to make some concessions from RL operations, and this does end up affecting Metro D's more than most, as they are the most vulnerable to having more local variations to procedures, which isn't suitable for VATSIM at times.

I apologise if I've requested changes on anything that's not your work, just trying to get it in to a state where we're all happy.